basic/atm/ATM solve .py

```diff
@@ -0,0 +1,95 @@
+__author__ = 'Shaked Manor'
+# takes customers information from file' then
+# creating objects from class customers with the customers information
+# takes input from user, checking his ID ans password and then
+# let the customer commit actions on his account
+
+CUSTOMERS_INFORMATION_FILE = r'c:\python\ATM.txt'
+END_VALUE = -1
+customer_passwords = {}
+customer_access_to_class = {}
+
+
+class customers:
+    """defining the actions that a customer can do in is account,
+     checking his balance, withdrawal and deposit money and change password"""
+
+    def __init__(self, password, balance):
+        self.password = password
+        self.balance = balance
+
+    def get_balance(self):
+        return self.balance
+
+    def cash_withdrawal(self):
+        withdrawal_amount = input('please enter the amount of money you want to withdrawal')
+        self.balance -= float(withdrawal_amount)
+        print f"the remainder of money in your account is: {self.balance}"
+
+    def cash_deposit(self):
+        deposit_amount = input('please enter the amount you want to deposit')
+        self.balance += float(deposit_amount)
+        print f"the remainder of money in your account is: {self.balance}"
+
+    def change_password(self):
+        y = input('please enter new password')
+        self.password = str(y)
+
+
+def users_input():
+    """takes users input, ID and password and compares it to the customer_password
+    dictionary to confirm its right. then takes the users wanted action and commit it"""
+    while True:
+        customer_input = input('please enter your customer ID')
+
+        if customer_input == END_VALUE:
+            print ('off')
+            break
+
+        else:
+            try:
+                customer_id = str(customer_input)
+                customer_access = customer_access_to_class[customer_id]
+
+            except:
```

Comment:
Dont catch all the exceptions in the world, it is prone to errors, be more specific.